modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -246,7 +246,7 @@ def update_status(status:, code: nil, detail: nil, raise_on_error: false)
             statusable_id: id,
             code:,
             detail:
-          }.stringify_keys
+          }.deep_stringify_keys
```

Comment:
It looks like the code in the actual `AppealReceivedJob` expects the nested keys to be stringified: https://github.com/department-of-veterans-affairs/vets-api/blob/5e50b7de609fefdf485cbcc11e30214e33eba1e8/modules/appeals_api/app/sidekiq/appeals_api/appeal_received_job.rb#L93-L98

Before, the nested email_identifier object had retained its original symbolized keys - now they will be strings as expected